boilerplate/frontend/supertokens-react-custom/src/hooks/useSessionInfo.ts

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from "react";
+import Session from "supertokens-web-js/recipe/session";
+
+function useSessionInfo() {
+    const [isUserLoggedIn, setIsUserLoggedIn] = useState(false);
```

Comment:
initial state should be undefined or loading, not false.

Reply:
changed this to
```jsx
const [isUserLoggedIn, setIsUserLoggedIn] = useState<undefined | boolean>(undefined);
```